build.gradle.kts

```diff
@@ -1,64 +1,85 @@
-  implementation(files("lib/org.eclipse.equinox.common-3.6.0.v20100503.jar"))
-  implementation(files("lib/org.eclipse.equinox.preferences-3.5.200-v20140224-1527.jar"))
-  implementation(files("lib/org.eclipse.jdt.ui-6.12.2.jar"))
-  implementation(files("lib/org.eclipse.jface.jar"))
-  implementation(files("lib/org.eclipse.ltk.core.refactoring-6.12.2.jar"))
-  implementation(files("lib/org.eclipse.osgi-3.7.1.jar"))
-  implementation(files("lib/org.eclipse.text_3.5.0.jar"))
-  // implementation(files("lib/snakeyaml-2.0.jar"))
-  // implementation(files("lib/sootclasses-trunk-jar-with-dependencies.jar"))
-  // implementation("org.apache.commons:commons-csv:1.5")
-  // implementation("commons-io:commons-io:2.16.1")
-  // implementation("org.eclipse.jdt:org.eclipse.jdt.core:3.40.0")
-  // implementation("org.eclipse.platform:org.eclipse.core.jobs:3.8.0")
-  // implementation("org.eclipse.core:org.eclipse.core.resources:3.7.100") // not the same
-  // implementation("org.eclipse.core:org.eclipse.core.runtime:3.7.0")
-  // implementation("org.eclipse.equinox:org.eclipse.equinox.common:3.6.0.v20100503")
-  // implementation("org.eclipse.platform:org.eclipse.equinox.preferences:3.6.1") // not the same
-  // implementation("org.eclipse.platform:org.eclipse.ltk.core.refactoring:3.14.600") // not the same
-  // implementation("org.eclipse.osgi:org.eclipse.osgi:3.7.1")
-  // implementation("org.eclipse.text:org.eclipse.text:3.5.101") // not the same
+  implementation("org.eclipse.jdt:org.eclipse.jdt.core:3.10.0.v20140604-1726")
+  implementation("org.apache.commons:commons-csv:1.5")
+  implementation("commons-io:commons-io:2.16.1")
+  implementation("org.eclipse.core:contenttype:3.4.200-v20140207-1251")
+  implementation("org.eclipse.core:org.eclipse.core.resources:3.7.100")
+  implementation("org.eclipse.core:org.eclipse.core.runtime:3.7.0")
+  implementation("org.eclipse.equinox:org.eclipse.equinox.common:3.6.0.v20100503")
+  implementation("org.eclipse.platform:org.eclipse.core.jobs:3.8.0")
+  implementation("org.eclipse.platform:org.eclipse.equinox.preferences:3.6.1")
+  implementation("org.eclipse.osgi:org.eclipse.osgi:3.7.1")
+  implementation("org.eclipse.text:org.eclipse.text:3.5.101")
   implementation("org.yaml:snakeyaml:2.0")
   implementation("org.soot-oss:soot:4.6.0")
+  testImplementation("org.junit.jupiter:junit-jupiter:5.11.3")
+  testImplementation("org.junit.jupiter:junit-jupiter-engine:5.11.3")
+  testImplementation("org.junit.jupiter:junit-jupiter-api:5.11.3")
   testImplementation("org.mockito:mockito-core:5.14.2")
-  testImplementation("org.testng:testng:7.10.2")
-  testImplementation("org.junit.jupiter:junit-jupiter-api:5.10.0")
+  testImplementation("org.testng:testng:7.5.1")
+}
+
+tasks.register("ide-paths") {
+  group = "Set-Up"
+  description = "Generate classpath file for IDEs to use for syntax and compiling"
+  doLast {
+    val classpathFile = file("my.classpath")
```

Comment:
Do we want this to generate `my.classpath`? or `.classpath`?

Reply:
I am worried about overwriting .classpath with out any sort of check so I would rather make a my.classpath file and let then rename it or copy what they want. I will change the wording to reflect this

Reply:
Ahh, that makes sense.